rllib/algorithms/appo/appo_learner.py

```diff
@@ -63,9 +68,10 @@ def additional_update_for_module(
         self,
         *,
         module_id: ModuleID,
+        hps: AppoLearnerHyperparameters,
```

Comment:
We are passing the module specific settings now into all `.._for_module` method calls. Strictly, this is not required as users could easily get these inside their implementations by doing:

```
def additional_update_for_module(self, module_id, ...):
   hps = self.hps.get_hps_for_module(module_id)
```
Not sure, might be cleaner this way, but I'd be also happy to remove this to keep the method signatures smaller.